source/2019-07-19-the-ember-times-issue-107.md

```diff
@@ -31,12 +32,13 @@ flexible imports with **Ember Auto Import** ⏬,
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [RFC: Major Version Release for All Deprecation Removal 📋](https://github.com/emberjs/rfcs/pull/512)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+An RFC was proposed by [@kerrick](https://github.com/Kerrick) to have a major version release for all deprecation removals. The main issue revolves around the [Intimate API](https://twitter.com/wycats/status/918644693759488005) policy which can catch some developers off guard when upgrading Ember versions.
+
+Since certain private APIs are considered "Intimate API" due to the wide range of addons and consumers who use it, despsite it being a private method we sometimes have issues when deprecating these private APIs. This is because not all private APIs would warrant a deprecation warning (and therefore a major version change). The RFC proposes to have a major version release for all deprecation removals.
```

Comment:
I think you can rephrase the first sentence for clarity.
```suggestion
Certain private APIs are considered "Intimate API" because there are addons and consumers who use them despite being private. We sometimes have issues when deprecating these private APIs. This is because not all private APIs would warrant a deprecation warning (and therefore a major version change). The RFC proposes to have a major version release for all deprecation removals.
```